ARImageTrackingExamples/ARImageTrackingExamples/View Controllers/SolidCardViewController.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+import UIKit
+import ARKit
+
+class SolidCardViewController: UIViewController, ARExperimentSessionHandler {
+
+    @IBOutlet var sceneView: ARSCNView!
+    private let arSessionDelegate = ARExperimentSession()
+    private let arViewModel = ARViewModel()
+
+    override func viewDidLoad() {
+        sceneView.delegate = self
+        arSessionDelegate.sessionHandler = self
+        sceneView.session.delegate = arSessionDelegate
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+
+        let configuration = arViewModel.imageTrackingConfiguration()
+        sceneView.session.run(configuration, options: [.removeExistingAnchors, .resetTracking])
+    }
+
+    override func viewWillDisappear(_ animated: Bool) {
+        sceneView.session.pause()
+    }
+}
+
+extension SolidCardViewController: ARSCNViewDelegate {
+
+    func renderer(_ renderer: SCNSceneRenderer, didAdd node: SCNNode, for anchor: ARAnchor) {
+        guard let imageAnchor = anchor as? ARImageAnchor else {
+            return
+        }
+
+        guard let image = arViewModel.image(correspondingTo: imageAnchor.referenceImage) else {
+            print("could not find image")
+            return
+        }
+        
+        let plane = SCNPlane(width: imageAnchor.imageSize().width,
+                             height: imageAnchor.imageSize().height)
+        plane.styleFirstMaterial(with: image)
+        let planeNode = SCNNode(geometry: plane)
+        planeNode.opacity = 0.95
+        planeNode.eulerAngles.x = -.pi / 2
+        planeNode.position.y = planeNode.position.y + 0.01
```

Comment:
Also I think is a good idea to use degrees instead of radians. We can add a static function (lets say a `degreesToRadians`) or a `Float` extension to do the conversion.

Reply:
or even use the [`Measurement` API](https://developer.apple.com/documentation/foundation/units_and_measurement)